### DISCUSSION TITLE
Server restart reductions (hotloading submarines, mod selections)

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#5480] [Ideas and suggestions]
2 similar purpose feature requests: 

For larger groups that are bigger/less tightly bonded than friends, restarting a server lobby can often mean massive loss of players, which sometimes are large enough that a game session can not continue as hoped. 

Add content package changes while in server lobby:
- player-made ships in /submarines/
- mods/content packages in /mods/ (settings)

doing both would greatly reduce the need for lobby restarts and make adding newly discovered lovelies more fluid to add.

**Comment by Leonidas-ai** [comment]
Yes please. Also, just reloading/exiting a server seems to cause a massive hang. Hopefully something can be done to make this process more efficient.
